### PR TITLE
Create events.cs-e2e.ts

### PR DESCRIPTION
**Pull request by cfaur09**
## Proposed Changes
- Create events.cs-e2e.ts

## How to test
- npm run test:e2e-cs.ts events.cs-e2e.ts

### CODE REVIEW

File: src/test/chain-simulator/events.cs-e2e.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      }
+
+      expect(event.topics).toBeInstanceOf(Array);
+    });
+
+    it('should support filtering by address', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events?address=${config.aliceAddress}`);
+      expect(response.status).toBe(200);
+      expect(response.data).toBeInstanceOf(Array);
+      expect(response.data.length).toBeGreaterThan(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/events.cs-e2e.ts:52]: maybe check that the events have the provided address

File: src/test/chain-simulator/events.cs-e2e.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    });
+
+    it('should support filtering by identifier', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events?identifier=completedTxEvent`);
+      expect(response.status).toBe(200);
+      expect(response.data).toBeInstanceOf(Array);
+      expect(response.data.length).toBeGreaterThan(1);
```
Comment (bogdan-rosianu) [inline src/test/chain-simulator/events.cs-e2e.ts:59]: maybe check for the identifier

File: src/test/chain-simulator/events.cs-e2e.ts (hunk 1 above)
Comment (cfaur09) [reply src/test/chain-simulator/events.cs-e2e.ts:52]: done

File: src/test/chain-simulator/events.cs-e2e.ts [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    });
+
+    // TODO: fix this
```
[the remark this answers is not in the document]
Comment (cfaur09) [reply src/test/chain-simulator/events.cs-e2e.ts:64]: I nee to chec this query, now it' returns [] even if txHash is valid. i will open a PR with proper fix, than I will unskip this test

File: src/test/chain-simulator/events.cs-e2e.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+  describe('GET /events', () => {
+    it('should return status code 200 and a list of events', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events`);
+      expect(response.status).toBe(200);
+      expect(response.data).toBeInstanceOf(Array);
+    });
+
+    it('should support pagination', async () => {
+      const response = await axios.get(`${config.apiServiceUrl}/events?from=0&size=2`);
```
Comment (dragos-rebegea) [inline src/test/chain-simulator/events.cs-e2e.ts:18]: we can add also a check that ?from=0&size=2 & ?from=1&size=2 returns different results

Comment (cfaur09) [reply src/test/chain-simulator/events.cs-e2e.ts:18]: done

**Review by bogdan-rosianu — APPROVED** [review]

**Review by dragos-rebegea — APPROVED** [review]

**Pull request merged**